### DISCUSSION TITLE
ADMX files for Winget-AutoUpdate-Intune

### DISCUSSION THREAD

**Discussion by 911TechGuy911** [Weatherlights/Winget-AutoUpdate-Intune#9] [Q&A]
Hi Tech Community!

I imported the ADMX and ADML files as supplied by this "Weatherlights / Winget-AutoUpdate-Intune" project,
however, when I then try to go and create an Intune Configuration policy, the relevant settings are not available under "Windows components". In-fact, even if I search for it, it's not there.

Import was successful.

Any idea why this would be?
Tried different tenants.

**Comment by Sambe00** [comment]
i get same issue. ADMX can uploaded but now show on configuration policys.

**Comment by Zaloxski** [comment]
Try this, when you completed the ADMX /ADML import, create a new intune profile: Templates->Imported Administrative templates (Preview)

**Reply by Eirinn75** [reply]
That's usually the correct place where to find all imported ADMX catalogs

**Reply by misterkent** [reply]
Thanks! Had issues finding it also.
